sync.py

```
@@ -0,0 +1,7 @@
+import aiohttp
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)